kytos/templates/kytos.conf.template

```diff
@@ -85,6 +85,14 @@ database =
 # MONGO_MIN_POOLSIZE=30
 # MONGO_TIMEOUTMS=30000
 
+# APM backend. Supported values: es (elasticsearch, don't use quotes in the string)
+apm =
+# The following environment variables should be accordingly if you're using elasticsearch:
+# ELASTIC_APM_URL=http://localhost:8200
+# Optionally, these environment variables can also be set:
+# ELASTIC_APM_SERVICE_NAME=kytos
+# ELASTIC_APM_SECRET_TOKEN=
```

Comment:
@italovalcy, yes. The rationale was that since APM and MongoDB have sensitive information that need to be defined, all of that are only supported via environment variables to avoid having these on the conf file, and in the conf file it's only specified which backend is being used. That way, as long as they are injected safely it should make it easier to deploy and to avoid leaking or storing credentials, like some cloud platforms typically do (with docker secrets loaded on env vars, or kubernetes secrets loaded on env vars). Let me know if you have other suggestions or preference on how to manage this, especially also considering what AmLight will likely use in prod.

Reply:
@italovalcy I'll go ahead and merge this, let's continue this thread discussion on issue #217, really appreciated your feedback and review.